data/json/mapgen/Pottery_Sewing_Shops.json

```diff
@@ -0,0 +1,121 @@
+[
+  {
+    "type": "mapgen",
+    "method": "json",
+    "om_terrain": [ "craft_shop" ],
+    "weight": 200,
+    "object": {
+      "fill_ter": "t_floor",
+      "rows": [
+        "____,____,____,____,____",
+        "____,____,____,____,____",
+        "____,____,____,____,____",
+        "____,____,____,____,____",
+        "____,____,____,____,____",
+        "ssssssssssssssssssssssss",
+        "ssssusssssssssssssusssss",
+        "s51|---+-|H;3H|-+---|15s",
+        "s11|.c..P-52;5-P..c.|11s",
+        "s4||TC..B|4554|n..CT||4s",
+        "s;|R.c..B||||||n..c.R|;s",
+        "s;|B....B|t|SS|n....B|;s",
+        "s;|B.nn..=wwww=..nn.n|4s",
+        "s;|B....B||||||n....n|;s",
+        "s4|B.....=L|L=......n|;s",
+        "s;|||-+-|||||||-+-||||4s",
+        "s;|E*******|QQQ***|nn|;s",
+        "s4|**nh*nh*|Q*****|=||;s",
+        "s;|**nh*nh*|Q*******q|;s",
+        "s;|E******q|Q*qY*Y**q|;s",
+        "s4|||||||=|||=||||||||4s",
+        "s;__UU_____4_____UU___;s",
+        "s;_________4__________4s",
+        "s;_________4__________;s"
+      ],
+      "terrain": {
+        ".": "t_floor",
+        "s": "t_sidewalk",
+        "u": "t_sidewalk",
+        "_": "t_pavement",
+        "U": "t_pavement",
+        ",": "t_pavement_y",
+        ";": "t_grass",
+        "H": "t_grass",
+        "1": "t_shrub_hydrangea",
+        "2": "t_tree_walnut",
+        "3": "t_tree_apple",
+        "4": "t_underbrush",
+        "5": "t_grass",
+        "-": "t_wall_glass",
+        "|": "t_brick_wall",
+        "+": "t_door_glass_c",
+        "=": "t_door_c",
+        "S": "t_linoleum_gray",
+        "w": "t_linoleum_gray",
+        "t": "t_linoleum_gray",
+        "c": "t_floor",
+        "C": "t_console_broken",
+        "P": "t_floor",
+        "T": "t_floor",
+        "B": "t_floor",
+        "n": "t_floor",
+        "R": "t_floor",
+        "L": "t_floor",
+        "*": "t_thconc_floor",
+        "Q": "t_thconc_floor",
+        "Y": "t_thconc_floor",
+        "A": "t_thconc_floor",
+        "q": "t_thconc_floor",
+        "E": "t_thconc_floor",
+        "h": "t_thconc_floor"
+      },
+      "furniture": {
+        "H": "f_bench",
+        "S": "f_sink",
+        "T": "f_stool",
+        "P": "f_indoor_plant",
+        "B": "f_bookcase",
+        "c": "f_counter",
+        "n": "f_table",
+        "h": "f_chair",
+        "L": "f_locker",
+        "R": "f_trashcan",
+        "Y": "f_clay_kiln",
+        "Q": "f_rack",
+        "q": "f_crate_c",
+        "E": "f_mannequin",
+        "U": "f_dumpster",
+        "5": "f_flower_tulip",
+        "u": "f_vending_reinforced"
+      },
+      "toilets": { "t": {  } },
+      "place_items": [
```

Comment:
In general i don't think a craft shop would be heavily looted. Since we're trying to change loot spawns in town to be more realistic, you could start with these PRs. I'd suggest you could increase the chance of item spawns to about 50-70 percent and put the repeats way up to 2-10 or so. Realistically, you'd probably find a craft shop very well stocked immediately after the apocalypse. That applies to all three maps... I doubt anyone was flocking to a glassblower's workshop to loot it at the end of the world.

Reply:
would you include workshop tool areas get the increased spawn rates

Reply:
Maybe a bit but not nearly as much